stop-es-cluster.sh

```diff
@@ -0,0 +1,12 @@
+kubectl config use-context elk
```

Comment:
And I agree with that thought. Problem here is that you're enforcing that to all users. And this is the only place where `elk` shows up which will add confusion.